src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+html {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  box-sizing: border-box;
```

Comment:
better to use `box-sizing` for `*` selector, it will apply styles to all elements